CHANGELOG.rst

```diff
@@ -4,6 +4,12 @@ Changelog
 in development
 --------------
 
+Added
+~~~~~
+
+* Install pack with the latest tag version if it exists when branch is not specialized.
+  (new feature) #354
```

Comment:
What's the reference URL behind the #354?

https://github.com/StackStorm/st2/pull/354 doesn't look relevant.

Reply:
> What's the reference URL behind the #354?
> 
> `https://github.com/StackStorm/st2/pull/354` doesn't look relevant.

https://github.com/StackStorm/discussions/issues/354

Reply:
Shortcuts like `#354` are transformed into #354 links automatically in Github referring to same repo issues or PRs.
Private discussion link that OpenSource community can't access doesn't look appropriate for the changelog, please include current PR reference instead.

See https://help.github.com/en/articles/autolinked-references-and-urls